webpack/dlls/vendor/webpack.config.js

```diff
@@ -0,0 +1,83 @@
+var path = require('path');
+var webpack = require('webpack');
+var root = path.resolve(__dirname, '..', '..', '..');
+
+module.exports = {
+  devtool: process.env.NODE_ENV === 'production' ? null : 'inline-source-map',
+
+  output: {
+    path: path.join(root, 'static/dist/dlls'),
+    filename: 'dll__[name].js',
+    library: 'DLL_[name]_[hash]'
+  },
+
+  entry: {
+    vendor: [
+      'babel-polyfill',
+
+      // <babel-runtime>
+      //
+      // Generate this list using the following command against the stdout of
+      // webpack running against the source bundle config (dev/prod.js):
+      //
+      //     egrep -o 'babel-runtime/\S+' | sed 's/\.js$//' | sort | uniq | wc -l
```

Comment:
This should be amended to include the full command to avoid the confusion (see main thread for context):

```shell
webpack \
  --config ./webpack/dev.config.js \
  --display-modules | egrep -o 'babel-runtime/\S+' | sed 's/\.js$//' | sort | uniq
```